lib/rabbitmq/cli/queues/commands/reclaim_quorum_memory_command.ex

```diff
@@ -0,0 +1,53 @@
+## This Source Code Form is subject to the terms of the Mozilla Public
+## License, v. 2.0. If a copy of the MPL was not distributed with this
+## file, You can obtain one at https://mozilla.org/MPL/2.0/.
+##
+## Copyright (c) 2007-2020 VMware, Inc. or its affiliates.  All rights reserved.
+
+defmodule RabbitMQ.CLI.Queues.Commands.ReclaimQuorumMemoryCommand do
+  alias RabbitMQ.CLI.Core.DocGuide
+
+  @behaviour RabbitMQ.CLI.CommandBehaviour
+  def scopes(), do: [:diagnostics, :queues]
```

Comment:
Does this really belong to `rabbitmq-diagnostics`?

Reply:
`rabbitmq-queues` ???

Reply:
I'd only expose this to `rabbitmq-queues`, yes.

Reply:
Agreed, might have been some copy & paste from the status command